recipe/meta.yaml

```diff
@@ -11,47 +11,68 @@ source:
   sha256: 4fd2eb3736cea503be94ab18d31a669addbaf2dd6a8249527cef94163bb93975
 
 build:
+  number: 4
   noarch: python
-  number: 3
-  script: python -m pip install . -vv
 
-requirements:
-  host:
-    - hatchling
-    - pip
-    - python {{ python_min }}
+outputs:
+  - name: osmnx-base
+    script: build_base.sh
```

Comment:
Does this need to be a separate file or can you just stick the one liner here?

Or can you just stick it in line 16 above (where you had removed it), to use it across the recipe rather than defining a different build script for each output?

Reply:
No, it seems that for multi-output receipts, a script file needs to be used. I tried with using the command directly and it failed.

Reply:
Is this still the case now that the multiple outputs have been approved?